[_]: feature/improve-shared-file-ux-ui

aboredvaro opened the pull request.

aboredvaro committed:
Updated UI

aboredvaro committed:
Added preview modal UI

aboredvaro committed:
Preview UI

aboredvaro committed:
Making Content accesible with keyboard

aboredvaro committed:
Fixed sidenav bug

aboredvaro committed:
WIP

aboredvaro committed:
WIP

aboredvaro committed:
Updated file preview

aboredvaro committed:
Preview message

aboredvaro committed:
Shared file before leave page prompt prepared

aboredvaro committed:
Minor design tweaks on file-list

aboredvaro committed:
WIP

aboredvaro committed:
Refactor from Class to Function

aboredvaro committed:
Added more Hooks and final touches

aboredvaro committed:
Delete ShareViewClass.tsx

aboredvaro committed:
Updated Cypress "Share File" spec

sg-gs commented on src/app/drive/components/DriveExplorer/DriveExplorer.scss:
```diff
@@ -9,21 +9,21 @@
-  ::-webkit-scrollbar {
-    width: 4px;
-  }
+  // ::-webkit-scrollbar {
```
If is not going to be used, could be removed instead of letting them there

sg-gs commented on src/app/drive/components/FileViewer/FileViewer.tsx:
```diff
@@ -11,9 +9,13 @@ import i18n from '../../../i18n/services/i18n.service';
 interface FileViewerProps {
   file: DriveFileData | null;
   onClose: () => void;
+  show: boolean;
```
As it seems like `show` prop is used for showing a transition, a better naming could be provided like: `showTransition` or `isLoading`

sg-gs commented on src/app/drive/components/FileViewer/FileViewer.tsx:
```diff
@@ -27,19 +29,18 @@ const extensionsList = fileExtensionService.computeExtensionsLists(fileExtension
+    setIsLoading: (value: boolean) => dispatch(fileViewerActions.setIsLoading(value))
   };
+
+  // console.log(props.file);
```
Remove `console.log` from here

sg-gs commented on src/app/referrals/components/ReferralsWidget/ReferralsWidget.scss:
```diff
@@ -44,8 +44,8 @@
-    &::-webkit-scrollbar {
-      width: 2px;
-    }
+    // &::-webkit-scrollbar {
```
dead code here too

sg-gs commented on src/app/tasks/components/TaskLogger/TaskLogger.scss:
```diff
@@ -1,25 +1,25 @@
+//   background-color: #c1c7d0;
+//   position: absolute;
+//   border: 15px solid white;
+//   border-radius: 100px;
```
to remove

sg-gs commented on src/index.scss:
```diff
@@ -229,25 +229,26 @@ code {
   input[type="checkbox"] {
-    @apply border-2 border-neutral-500 rounded-1px text-xs text-neutral-500 w-3 h-3;
+    @apply border-2 border-neutral-500 rounded text-xs text-neutral-500 w-4 h-4;
+    // @apply appearance-none border border-neutral-50 rounded w-4 h-4 bg-neutral-10 bg-opacity-50 checked:bg-blue-60 checked:border-blue-60;
```
remove this

sg-gs reviewed (changes requested):
Hey @aboredvaro, thanks for contributing, please consider from here in advance to contribute with small PRs. ([check this Confluence section](https://inxt.atlassian.net/wiki/spaces/T/pages/13762589/Contributing+guidelines) dedicated to contributing guidelines).

Thus, you can have your PR reviewed faster and better. Also if something has to be rollbacked, we do not have to undo 93 files changed + would be easier to spot, find and fix the error.

aboredvaro committed:
Applied all Sergio's suggestions

-  Removed all commented code
-  Removed dead code and styles
-  Provided better naming for ambiguous variables
